data/net/minecraft/advancements/critereon/EntityHurtPlayerTrigger.mapping

```diff
@@ -3,15 +3,22 @@ CLASS net/minecraft/advancements/critereon/EntityHurtPlayerTrigger
 		ARG 1 json
 		ARG 2 entityPredicate
 		ARG 3 conditionsParser
+	METHOD lambda$trigger$0 (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/damagesource/DamageSource;FFZLnet/minecraft/advancements/critereon/EntityHurtPlayerTrigger$TriggerInstance;)Z
+		ARG 5 instance
 	METHOD trigger (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/damagesource/DamageSource;FFZ)V
 		ARG 1 player
 		ARG 2 source
 		ARG 3 amountDealt
 		ARG 4 amountTaken
 		ARG 5 wasBlocked
 	CLASS TriggerInstance
+		METHOD <init> (Lnet/minecraft/advancements/critereon/EntityPredicate$Composite;Lnet/minecraft/advancements/critereon/DamagePredicate;)V
+			ARG 1 player
+			ARG 2 damage
 		METHOD entityHurtPlayer (Lnet/minecraft/advancements/critereon/DamagePredicate$Builder;)Lnet/minecraft/advancements/critereon/EntityHurtPlayerTrigger$TriggerInstance;
 			ARG 0 damageConditionBuilder
+		METHOD entityHurtPlayer (Lnet/minecraft/advancements/critereon/DamagePredicate;)Lnet/minecraft/advancements/critereon/EntityHurtPlayerTrigger$TriggerInstance;
+			ARG 0 damage
 		METHOD matches (Lnet/minecraft/server/level/ServerPlayer;Lnet/minecraft/world/damagesource/DamageSource;FFZ)Z
```

Comment:
These names are inconsistent with `DamagePredicate`, should be:
`dealtDamage`
`takenDamage`
`blocked`